### ISSUE TITLE
Wrong checker in ...\listings\pb\pb6server.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11456]
|<img alt="Klamann's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Klamann reported|
|-|-|
|Trac ID|trac#386|
|Type|defect|
|Created|2003-11-07 01:16:55Z|

<details><summary>Searchable metadata</summary>

```
trac-id__386 386
type__defect defect
reporter__Klamann Klamann
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1068167815000000 1068167815000000
changetime__1068186540000000 1068186540000000
version__ 
owner__ 
cc__Klamann
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Klamann's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Klamann commented|
|-|-|

```
#!html
<pre>
In the example program exists a small error :
the checker 'checkers.InMemoryDatabaseDontUse' does not exist. Instead it should 
read 'checkers.InMemoryUsernamePasswordDatabaseDontUse'
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Fixed in CVS revision 1.5.  Thanks!
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
